spec/bnc/_A_Express_checkout_flow.test.js

```diff
@@ -25,5 +25,7 @@ describe.only('Wine Sub Checkout', function() {
         browser.setValue('#cvv', '234');
         browser.moveToObject('.terms-and-conditions-right-col .btn--primary');
         browser.click('.terms-and-conditions-right-col .btn--primary');
+        browser.waitUntil(1000);
```

Comment:
Rather than waiting for 1 sec here, wait for a unique element on next page. This may fail if page does not load in 1 sec.